AutomatedTesting/Gem/PythonTests/editor_test_testing/EditorTest_That_PassesToo.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Contributors to the Open 3D Engine Project
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+import time
+
+def EditorTest_That_PassesToo():
+    pass
```

Comment:
question: Can you not run/invoke the ```EditorTest_That_Passes``` twice instead of duplication?

Reply:
What really is the usefulness of this test?
What added value does it provide than just running the `editor_python_test_tools` suite with a normal test?

Reply:
These tests test that fail, crash and succeed do what is expected. We can’t do this in normal tests because it’s a test of the testing framework

Reply:
This way we avoid tests passing when they are not supposed to